suite-native/app/e2e/tests/deeplinkPopup.test.ts

```diff
@@ -0,0 +1,162 @@
+import http from 'http';
+import { exec } from 'child_process';
+
+import { TrezorUserEnvLink } from '@trezor/trezor-user-env-link';
+import TrezorConnect from '@trezor/connect-mobile';
+
+import {
+    appIsFullyLoaded,
+    disconnectTrezorUserEnv,
+    openApp,
+    prepareTrezorEmulator,
+    restartApp,
+} from '../utils';
+import { onOnboarding } from '../pageObjects/onboardingActions';
+import { onCoinEnablingInit } from '../pageObjects/coinEnablingActions';
+import { onHome } from '../pageObjects/homeActions';
+
+const SERVER_PORT = 8080;
```

Comment:
Is there any chance that I would be running this locally while having connect-explrorer dev server active? I think that we are using this port somewhere already. clash is probably unlikely however.

Reply:
As far as I know connect-explorer is running on port 8088.

Reply:
right, I saw it here https://github.com/trezor/trezor-suite/blob/develop/docs/features/coinjoin.md#L15